mds_loadtest/scripts/web_locust_file.py

```diff
@@ -0,0 +1,87 @@
+from locust import HttpLocust, TaskSet, seq_task, task
+import constants
+
+BEARER_TOKEN_STRING ="Bearer " + constants.BEARER_TOKEN
+MINE_GUID = constants.MINE_GUID
+class UserBehavior(TaskSet):
+
+    def on_start(self):
+        """ on_start is called when a Locust start before any task is scheduled """
+        print("Started")
+
+    def on_stop(self):
+        """ on_stop is called when the TaskSet is stopping """
+        print("Stopped")
+
+    @task(1)
+    def home(self):
+        self.client.get("/",headers={"Authorization": BEARER_TOKEN_STRING})
+
+    #Perform search and filters
+    @task(1)
+    def search(self):
+        self.client.get("/dashboard?page=1&per_page=25&search=apple&commodity=&region=&status=&tenure=",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+    @task(1)
+    def search_tenure_filter(self):
+        self.client.get("/dashboard?page=1&per_page=25&search=&commodity=&region=&status=&tenure=COL",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+
+    @task(1)
+    def search_status_filter(self):
+        self.client.get("/dashboard?page=1&per_page=25&search=&commodity=&region=&status=ABN&tenure=",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+
+    @task(1)
+    def search_region_filter(self):
+        self.client.get("/dashboard?page=1&per_page=25&search=&commodity=&region=SC&status=&tenure=",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+
+    @task(1)
+    def search_commodity_filter(self):
+        self.client.get("/dashboard?page=1&per_page=25&search=&commodity=TO&region=&status=&tenure=",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+
+    @task(1)
+    def search_region_filter(self):
+        self.client.get("/dashboard?page=1&per_page=25&search=&commodity=&region=SC&status=&tenure=",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+
+    @task(1)
+    def search_major_filter(self):
+        self.client.get("/dashboard?page=1&per_page=25&major=true&commodity=&region=&status=&tenure=",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+
+    @task(1)
+    def search_tsf_filter(self):
+        self.client.get("/dashboard?page=1&per_page=25&tsf=true&commodity=&region=&status=&tenure=",
+                        headers={"Authorization": BEARER_TOKEN_STRING})
+
+    #Test map response
+    @task(1)
+    def map(self):
+        self.client.get("/dashboard?page = 1 & per_page = 25 & map = true",
```

Comment:
Does this actually work with all of those extra spaces? Are they there for a reason?

Reply:
yes, but I fixed